transaction/core/tests/validation.rs

```diff
@@ -518,7 +520,7 @@ fn test_validate_signature_ok() {
 #[test]
 // Should return InvalidTransactionSignature if an input is modified.
 fn test_transaction_signature_err_modified_input() {
-    let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
+    let mut rng: RngType = SeedableRng::from_seed([1u8; 32]);
```

Comment:
If we're changing these lines, how about we use `mc_util_test_helper::get_seeded_rng()`?

Reply:
yeah i was trying to see if I could break dependency on `rand` crate in this crate, was almost able to do it